### PR TITLE
fix: sqlite provider import sub doc db file

### PR DESCRIPTION
**Pull request by pengx17**
In the new migration workflow, workspace persisted in the electron cache path will be removed and re-created as well via `bootstrap`.

One missing part is that how to due with import the old DB. In this pr, we prevent user from loading old format db files when validating.

### CODE REVIEW

**Comment by pengx17** [comment]
@Brooooooklyn @himself65 please take a look if this approach is acceptable.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -352,14 +352,8 @@ jobs:
...
           NATIVE_TEST: 'true'
 
-      - name: Build Infra
-        run: yarn run build:infra
-
-      - name: Build Plugins
-        run: yarn run build:plugins
-
       - name: Build Desktop Layers
-        run: yarn workspace @affine/electron build
```
Comment (himself65) [inline .github/workflows/build.yml]: replace all ci actions

**Review by himself65 — CHANGES_REQUESTED** [review]
Is it possible to test the migration in e2e? Moving a `.affine` file into fixtures is acceptable.

**Review by himself65 — APPROVED** [review]

**Comment by himself65** [comment]
hope everything works!

**Pull request merged**